data/base-town.yaml

```diff
@@ -31,10 +31,6 @@ Crossing:
   gemshop:
     id: 4652
     name: appraiser
-  pawnshop:
```

Comment:
Why was this removed?

Reply:
:-/

Because I added those outpost rooms on a different computer and didn't pay close enough attention to the diff.  I'll roll those back tomorrow when I'm back home.

Reply:
The pawnshops have been replaced, and thank you for catching that.